modules/wear_detect.js

```diff
@@ -11,9 +11,7 @@ exports.isWorn = function() {
   return new Promise(resolve => {
     if (Bangle.isCharging())
         return resolve(false);
-    if (E.getTemperature() > 24.625)
-        return resolve(true);
-    if (Bangle.getAccel().mag > 1.045)
+    if (Bangle.getHealthStatus().movement > 124)
```

Comment:
Actually worth noting that the docs mention `movement is the 32 bit sum of all acc.diff readings since power on` - so this reading keeps going up. If you call this right after the 10 minute timeslot changes it'll be 0, but 9 minutes after it'll be 9x bigger than the value 1x after - I think this line needs a bit more thought.